src/libstd/sys/unix/os.rs

```diff
@@ -513,3 +513,7 @@ pub fn home_dir() -> Option<PathBuf> {
 pub fn exit(code: i32) -> ! {
     unsafe { libc::exit(code as c_int) }
 }
+
+pub fn getpid() -> u32 {
+    unsafe { libc::getpid() as u32 }
```

Comment:
`pid_t` is technically signed by standard, defined as `i32` on all of `libc`'s unix platforms.

Reply:
Yes, but `getpid()` is guaranteed to return a non-negative number, and so `u32` is valid here.